authoring-new-actions.md

```diff
@@ -0,0 +1,90 @@
+
+- [actions-admin](https://github.com/orgs/Azure/teams/actions-admin/members) team would be added with "admin" role to every new GH repo created for authoring any new action under the Azure org (github.com/azure/)
+- Owners of the individual actions could manage the "maintainer" permissions for their teams by [creating a new github team handle](https://github.com/orgs/Azure/new-team) for their crews and adding this handle with "admin"/"maintainer" role as needed
+- Kindly request the corresponding Azure product teams to include their team handle as "Admins"/"Maintainers" to facilitate raising any commits from their side for feature enhancements or for fixing supportability issues
+
+## Guidelines during Development phase
+
+<Link to complete guidance during development>
+
+## ReadMe.md &amp; action.yml files
+
+Once the Action implementation is complete, ensure that it is documented well in the readme.md file at the root of the repo of the Master branch. Please note that this content is used when the Action is published to the GitHub Marketplace. Hence including a sample workflow which covers the best usage of the Action is always recommended.
+
+Also ensure that the Action.yml is present in the root of the repo and has the Action name &amp; description well defined. Also choose the icon &amp; color suitably under branding as these are used when the Action is listed in Marketplace catalog.
+
+Note: All actions published from **Azure** GitHub org are branded using the Azure icon irrespective of the icon used in the action.yml. This is to leverage the recall value of Azure brand and icon as compared to the individual icons of various Azure services. 
+
+## Action Versioning
+
+**Versioning Guidance for GH actions that should be adhered to before a new action or an enhancement to an existing action is released:** 
+
+Official Release Management Guidance from GH platform team: https://docs.github.com/en/free-pro-team@latest/actions/creating-actions/about-actions#using-release-management-for-actions
+
+**Note:** Introduce a new major version of the branch (releases/v2) and tag (v2) if changes break the existing workflow. Ensure to move the major version tag to always point to the most recent minor version tag to ensure all the existing users of the major version would automatically gain the benefits from the new enhancements added in the minor version release.
+
+
+## Publish the action to Marketplace
+
+### Pre-requisites:
+
+- Ensure that the Legal OSS approval process for the repo is complete. Search for the status of the work item created with your repo name here: https://ossmsft.visualstudio.com/Reviews/\_workitems/ .
+
+&quot;DO NOT PUBLISH THE ACTION TO THE MARKETPLACE UNTIL LEGAL REVIEW IS COMPLETE&quot;
+
+- Versioning guidelines mentioned above section are adhered to and you have a version tag
+- Have a readme.md in place with documentation
+- Include V-team for review process and sign off by mailing@ [V team Alias](mailto:)
```

Comment:
Please add the team name and alias instead of the "V-team" placeholder

Reply:
done